test/jdk/java/awt/color/ICC_ColorSpace/ToFromCIEXYZRoundTrip.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2022 ?

Reply:
Yes, that code seems related to the KCMS. I'll update the date.